scheduler/docs/configuration.asc

```diff
@@ -172,6 +164,36 @@ The Cook scheduler comes with a few knobs to tune its behavior under the `:sched
   This option is a map that allows you to configure limits for tasks, to ensure that impossible-to-schedule tasks and tasks that run forever won't bog down your cluster.
   It currently supports 4 parameters to defend the Cook scheduler, which are described in <<task_constraints>>.
 
+
+[[rebalancer]]
+==== Rebalancer configuration
+
+Optionally, you can include a "rebalancer" stanza.  If you do, on startup, Cook will update its link:rebalancer-config.asc[Rebalancer configuration] to match the values you specify here.
+
+ :interval-seconds::
+   How often to rebalance the cluster for fairness between users.  Default is 300 (5 minutes).
+
+ :safe-dru-threshold::
+  See the link:rebalancer-config.asc[Rebalancer documentation]
+
+ :min-dru-diff::
+  See the link:rebalancer-config.asc[Rebalancer documentation]
+
+ :max-preemption::
+  See the link:rebalancer-config.asc[Rebalancer documentation]
+
+ :min-utilization-threshold::
+  See the link:rebalancer-config.asc[Rebalancer documentation]
+
+`:dru-scale`::
+  This is only used to control the metrics reporting of DRU values.  On some clusters,
+  the DRU's may be so small that when the values are fed to clj-metrics, they are
+  treated as 0, which makes it impossible to glean insights into the DRU's in play,
+  in order to set rebalancer parameters.
+  If you find that this is true on your cluster, try increasing the dru-scale setting
+  to e.g. 10^300.
```

Comment:
Instead of encouraging them to change this scale, encourage them to look at their settings for share, it is likely they are max_double